src/data/script.ts

```diff
@@ -247,8 +247,12 @@ export interface StopAction extends BaseAction {
   error?: boolean;
 }
 
+export interface SequenceAction extends BaseAction {
+  sequence: (ManualScriptConfig | Action)[];
+}
+
 export interface ParallelAction extends BaseAction {
-  parallel: ManualScriptConfig | Action | (ManualScriptConfig | Action)[];
+  parallel: SequenceAction[];
```

Comment:
It should still accept config that is not in a list right?

Reply:
Sorry, this was forgotten there and is now fixed.

Reply:
Does this look good now @bramkragten , or do I need to fix anything more?